.github/workflows/merge-up.yml

```diff
@@ -0,0 +1,22 @@
+name: Merge-up
+
+on:
+  # At 4:20 AM UTC, only on Sunday
+  schedule:
+    - cron: '20 4 * * 0'
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  merge-up:
+    name: Merge-up
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
```

Comment:
```suggestion
    if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
```??

Reply:
This is intentionally `silverstripe` to cover the case when this module is transferred to the officially supported modules. For now the condition is expected to fail.